src/lib/Containers/Inbox.tsx

```diff
@@ -1,29 +1,47 @@
 import * as React from "react"
 import * as Relay from "react-relay"
+import styled from "styled-components/native"
 
-import { ScrollView } from "react-native"
+import { ScrollView, View } from "react-native"
 import ActiveBids from "../Components/Inbox/Bids"
 import Conversations from "../Components/Inbox/Conversations"
 import ZeroStateInbox from "../Components/Inbox/Conversations/ZerostateInbox"
 
-export class Inbox extends React.Component<any, any> {
-  currentScrollOffset?: number = 0
+interface State {
+  hasBids?: boolean
+  hasMessages?: boolean
+}
+
+const Container = styled.View`
+  flex: 1;
+`
+
+export class Inbox extends React.Component<any, State> {
+  constructor(props) {
+    super(props)
 
-  hasNoContent() {
-    const userHasConversations = this.props.me.conversations.edges.length > 0
-    return !userHasConversations
+    this.state = {
+      hasBids: false,
+      hasMessages: false,
+    }
   }
 
   render() {
-    return this.hasNoContent()
-      ? <ZeroStateInbox />
-      : <ScrollView
-          onScroll={event => (this.currentScrollOffset = event.nativeEvent.contentOffset.y)}
-          scrollEventThrottle={10}
-        >
-          <ActiveBids me={this.props.me} />
-          <Conversations me={this.props.me} />
-        </ScrollView>
+    const updateBidsState = hasBids => {
+      this.setState({ hasBids })
+    }
+    const updateMessagesState = hasMessages => {
+      this.setState({ hasMessages })
+    }
+
+    const shouldShowEmptyState = !this.state.hasBids && !this.state.hasMessages
+    const headerView = <ActiveBids me={this.props.me} onDataLoaded={updateBidsState} />
+    return (
+      <Container>
+        <Conversations me={this.props.me} headerView={headerView} onDataLoaded={updateMessagesState} />
```

Comment:
@alloy @mennenia `onDataLoaded` is not defined on the `Conversations` relay container. When I use `RelayProps` as the props interface for the `Inbox` component. My guess is the typescript definition of relay container don't expose other properties aside from fragments but I'd have to confirm if that's actually the case.

Here's the error:
```
[ts]
Type '{ me: {}; onDataLoaded: (hasBids: any) => void; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Component<Props, ComponentState>> & Readonly<{ chi...'.
  Type '{ me: {}; onDataLoaded: (hasBids: any) => void; }' is not assignable to type 'Readonly<Props>'.
    Types of property 'me' are incompatible.
      Type '{}' is not assignable to type '{ lot_standings: { active_bid: { __id: string; }; }[]; }'.
        Property 'lot_standings' is missing in type '{}'.
```
 
<img width="1440" alt="screen shot 2017-06-30 at 3 31 24 pm" src="https://user-images.githubusercontent.com/296775/27737651-40d458e8-5da9-11e7-9170-755620ec2677.png">